raiden/tests/conftest.py

```diff
@@ -143,6 +143,8 @@ def logging_level(request):
         colorize=not request.config.option.plain_log,
         log_file=request.config.option.log_file,
         cache_logger_on_first_use=False,
+        # Prevent duplication of logging into multiple log handlers
+        disable_debug_logfile=True,
```

Comment:
Left some description of what i investigated to try to reproduce the `Captured log`. Although not 100% certain that this marked line will fix the issue, i noticed that this is the only code path that will enable 2 handlers (Default + debug-log) for logging which means that each handler will output the same log message into their designated destination.